### PR TITLE
chore: Add troubleshooting section to install guide

### PR DESCRIPTION
**Pull request by peblair**

### CODE REVIEW

File: src/getting_grain.md [hunk 1]
```diff
@@ -82,6 +82,20 @@ You can check that everything is installed properly by running the version comma
 grain --version
 ```
 
-If you see a a version for the CLI and the compiler, you're all set!
+If you see a a version for the CLI and the compiler, you're all set! If not, please consult the [Troubleshooting](#Troubleshooting) section.
 
 We'll first use the `grain` CLI to compile and run a Hello World program, but first let's set up our editor.
+
+## Troubleshooting
```
Comment (ospencer) [inline src/getting_grain.md:89]:
```suggestion
### Troubleshooting
```

File: src/getting_grain.md [hunk 2]
```diff
@@ -82,6 +82,20 @@ You can check that everything is installed properly by running the version comma
...
+
+This section contains solutions to problems which people may run into when building and installing Grain.
+
+### After install, the `grain` command is not found
```
Comment (ospencer) [inline src/getting_grain.md:93]:
```suggestion
#### After install, the `grain` command is not found
```

**Review by ospencer — COMMENTED** [review]
Looks great! Funny enough, I might prioritize the instructions for `zsh` over `bash` since more developers have Macs.

**Review by marcusroberts — APPROVED** [review]

**Review by ospencer — APPROVED** [review]
Looks great! Great workshopping!

**Pull request merged**